packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs

```diff
@@ -81,8 +82,8 @@ export class <%= className %>Controller {
       },
     },
   })
-  async findById(@param.path.<%= idType %>('id') id: <%= idType %>): Promise<<%= modelName %>> {
-    return await this.<%= repositoryNameCamel %>.findById(id);
+  findById(@param.path.<%= idType %>('id') id: <%= idType %>): Promise<<%= modelName %>> {
```

Comment:
I'm confused about the need for this change. All other controller methods are still using async/await. Why does this one need to be treated differently?

Reply:
+1 to be consistent.

I am going to revert this change and keep the original `async` method with `return await` statement.